ui/src/shared/copy/notifications.js

```diff
@@ -406,6 +412,27 @@ export const notifyDashboardDeleted = name => ({
 export const notifyDashboardDeleteFailed = (name, errorMessage) =>
   `Failed to delete Dashboard ${name}: ${errorMessage}.`
 
+export const notifyCellAdded = () => ({
+  ...defaultSuccessNotification,
+  icon: 'dash-h',
+  duration: 2200,
+  message: 'Added "Untitled Cell" to dashboard.',
+})
+
+export const notifyCellCloned = name => ({
+  ...defaultSuccessNotification,
+  icon: 'duplicate',
+  duration: 2200,
+  message: `Added "${name}" to dashboard.`,
```

Comment:
name here is the name of to the original cell, maybe clone message should say``` Added "${name}(clone)" to dashboard.``` ? or ```Cloned "${name}" to dashboard.```?

Reply:
The cloned cell already as `(Clone)` appended to the name at this point, so it seemed redundant